pom.xml

```diff
@@ -34,6 +34,10 @@
 
     <properties>
         <air.test.jvmsize>2g</air.test.jvmsize>
+        <project.build.targetJdk>17</project.build.targetJdk>
+        <air.java.version>17.0.4</air.java.version>
```

Comment:
why not 17.0.5?

Reply:
This doesn't matter, does it?

replaced with just `17`